cmd/main.go

```diff
@@ -36,10 +36,10 @@ import (
 	"github.com/IBM/ibm-csi-common/pkg/metrics"
 	mountManager "github.com/IBM/ibm-csi-common/pkg/mountmanager"
 	"github.com/IBM/ibm-csi-common/pkg/utils"
-	"github.com/IBM/ibm-csi-common/pkg/watcher"
 	csiConfig "github.com/IBM/ibm-vpc-file-csi-driver/config"
 	driver "github.com/IBM/ibm-vpc-file-csi-driver/pkg/ibmcsidriver"
-	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/ibmcloudprovider"
+	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/pkg/ibmcloudprovider"
+	"github.com/IBM/ibmcloud-volume-file-vpc/pkg/watcher"
```

Comment:
Do we really need watcher ? We can discuss with @arahamad once. If tagging is working we should avoid this. as of now let it be there

Reply:
We can avoid it and let it be used from `ibm-csi-common` but then will get references of `ibmcloud-volume-vpc` in go.mod

Reply:
Ok got it , I am talking about removing watcher usage all together for file. We can discuss.